### ISSUE TITLE
Extend ParentRunner from Sputnik

### ISSUE THREAD

**Issue by chrylis** [spockframework/spock#63]
Maven's Surefire and Failsafe plugins provide single-JVM parallel test processing that can be very helpful with high-latency Selenium tests, [but they need the runner to extend `ParentRunner` in order to use thread-based parallelism](http://maven.apache.org/surefire/maven-failsafe-plugin/examples/fork-options-and-parallel-execution.html). `Sputnik` extends `Runner` directly, and I believe I'm not being able to use JVM-level parallel runs because of that. Would it be practical at all for `Sputnik` to extend `ParentRunner`, or would the Spock magic conflict with that?

**Comment by Fuud** [comment]
I am currently working on it (because our integration tests became too slow - we need to parallel them).
Pull request will be performed before end of March (as I estimate).

**Comment by Fuud** [comment]
@chrylis I submitter pull request #67.

You can try builded version by downloading
https://github.com/Fuud/mvn-repo/raw/master/releases/org/spockframework/spock-core-fuud-fork/1.0.1-groovy-2.3/spock-core-fuud-fork-1.0.1-groovy-2.3.jar
or using maven
```
    <repositories>
        <repository>
            <id>fuud.forks</id>
            <name>Fuud Forks</name>
            <url>https://raw.githubusercontent.com/Fuud/mvn-repo/master/releases</url>
            <releases>
                <enabled>true</enabled>
                <updatePolicy>never</updatePolicy>
            </releases>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
        </repository>
    </repositories>

    <dependencies>
          ...
        <dependency>
            <groupId>org.spockframework</groupId>
            <artifactId>spock-core-fuud-fork</artifactId>
            <version>1.0.1-groovy-2.3</version>
            <scope>test</scope>
        </dependency>
    </dependencies>
```

Your feedback is highly appreciated.

**Comment by michaelsalvia** [comment]
@Fuud It appears I am unable to access your repository location.

**Comment by leonard84** [comment]
Closing since it was implemented by @Fuud with #67.
